packages/shared/src/domain/patient/patient-import.ts

```diff
@@ -13,7 +13,7 @@ export const patientImportPatientSchema = z.object({
   lastname: z.string().min(1, { message: "Last name must be defined" }),
   zip: z.string().refine(arg => normalizeZipCodeNewSafe(arg), { message: "Invalid zip" }),
   city: z.string().min(1, { message: "City must be defined" }),
-  state: z.string().refine(normalizeStateSafe, { message: "Invalid state" }),
+  state: z.string().refine(normalizeUSStateForAddress, { message: "Invalid state or territory" }),
```

Comment:
@thomasyopes, this was using the safe version (returns undefined), but it has a message for invalid value - so I used the strict version of the implementation.

Reply:
The idea behind safe was `refine` expects a function that returns a truthy value. If we literally throw an error here, will be lose the overall zod repsonse?

Reply:
done